modules/provision/provision.go

```diff
@@ -27,26 +27,19 @@ const (
 // ReplyTo defines how report the result of the provisioning operation
 type ReplyTo string
 
-// Tenant defines the tenant identity
-type Tenant string
-
-func (t Tenant) String() string {
-	return string(t)
-}
-
 // Reservation struct
 type Reservation struct {
 	// ID of the reservation
 	ID string `json:"id"`
-	// Tenant ID
-	Tenant Tenant `json:"tenant"`
-	// ReplyTo is a dummy attribute to hold the 3bot address
-	// we need to report to once the reservation is done
-	ReplyTo ReplyTo `json:"reply-to"`
+	// Identification of the user requesting the reservation
+	User string `json:"user_id"`
```

Comment:
Why the `reply-to` was dropped? we still need to notify the user `3bot` on the reservation result. Do u have another idea to do this?

Reply:
This is not an information that needs to be in the reservation. We already have the userID (3bot) from which you get the current address of this 3bot. This address might change so no point having it in the reservation